addons/grenades/CfgAmmo.hpp

```diff
@@ -119,6 +119,16 @@ class CfgAmmo {
         effectsSmoke = "ACE_M84FlashbangEffect";
         whistleDist = 0;
     };
+    
+    class ACE_G_CTS9: ACE_G_M84 {
+        GVAR(flashbang) = 1;
+        GVAR(flashbangBangs) = 9;
+        GVAR(flashbangInterval) = 0.5;
+        GVAR(flashbangIntervalStandardDeviation) = 0.1;
```

Comment:
```suggestion
        GVAR(flashbangIntervalMaxDeviation) = 0.35;
```
The variable was renamed to max deviation and now corresponds to the half width of the n=4 Bates distribution, so this is equivalent via `0.1 * sqrt (12*4) / 2  ≈ 0.35`.